CONTRIBUTING.md

```diff
@@ -4,28 +4,53 @@ Cisco Network Elements support a rich set of features to make networks robust, e
 ## Getting Started
 
 * Create a [GitHub account](https://github.com/signup/free)
-* Create a [cisco.com](http://cisco.com) account if you need access to a Network Simulator to test your code.
+* N9kv. A virtual Nexus N9000/N3000 may be helpful for development and testing. Users with a valid [cisco.com](http://cisco.com) user ID can obtain a copy of a virtual Nexus N9000/N3000 by sending their [cisco.com](http://cisco.com) user ID in an email to <get-n9kv@cisco.com>. If you do not have a [cisco.com](http://cisco.com) user ID please register for one at [https://tools.cisco.com/IDREG/guestRegistration](https://tools.cisco.com/IDREG/guestRegistration)
```

Comment:
Looks a bit odd:

* N9kv. A virtual Nexus N9000/N3000 may...

Should the 'N9kv. ' be removed?